inference-engine/tests/functional/inference_engine/ngraph_reader/deformable_psroi_pooling_tests.cpp

```diff
@@ -121,5 +121,124 @@ TEST_F(NGraphReaderTests, ReadDeformablePSROIPoolingNetwork) {
 	</edges>
 </net>
 )V0G0N";
-    compareIRs(model, modelV7, 117600);
-}
\ No newline at end of file
+    compareIRs(model, modelV7);
```

Comment:
Should it not fail if we provide an incorrect attribute?

Reply:
As I wrote in the description, it doesn't fail. So additional check for mode attribute would break backward compatibility.
The test was incorrect, but it proves that with the current version it is possible to create DeformablePSROIPooling with modes supported by PSROIPooling and handle it without errors by conversion to legacy.